packages/discussions/src/types.ts

```diff
@@ -208,13 +208,22 @@ export enum PostSort {
 // mixins
 
 /**
- * authoring properties
+ * author property
  *
  * @export
  * @interface IWithAuthor
  */
 export interface IWithAuthor {
```

Comment:
I _believe_ we also need to update `IUpdateChannel` to add the `author` property.

```typescript
/**
 * dto for updating channel settings
 *
 * @export
 * @interface IUpdateChannel
 * @extends {Partial<IWithSettings>}
 */
export interface IUpdateChannel extends Partial<IWithSettings>, Partial<IWithAuthor> {}
```

Otherwise clients won't be able to pass an `author` property value to [updateChannel](https://github.com/Esri/hub.js/blob/splitting-authoring-interfaces/packages/discussions/src/channels.ts#L62)